velox/substrait/SubstraitToVeloxPlan.cpp

```diff
@@ -180,6 +223,49 @@ core::PlanNodePtr SubstraitVeloxPlanConverter::toVeloxPlan(
       childNode);
 }
 
+core::PlanNodePtr SubstraitVeloxPlanConverter::toVeloxPlan(
+    const ::substrait::FetchRel& fetchRel) {
+  core::PlanNodePtr childNode;
+  // Check the input of fetchRel, if it's sortRel, convert them into
+  // topNNode. otherwise, to limitNode.
+  ::substrait::SortRel sortRel;
+  bool topNFlag;
+  if (fetchRel.has_input()) {
+    topNFlag = fetchRel.input().has_sort();
+    if (topNFlag) {
+      sortRel = fetchRel.input().sort();
+      childNode = toVeloxPlan(sortRel.input());
+    } else {
+      childNode = toVeloxPlan(fetchRel.input());
+    }
+  } else {
+    VELOX_FAIL("Child Rel is expected in FetchRel.");
+  }
+
+  if (topNFlag) {
+    auto [sortingKeys, sortingOrders] =
+        processSortField(sortRel.sorts(), childNode->outputType());
+
+    VELOX_CHECK_EQ(fetchRel.offset(), 0);
+
+    return std::make_shared<core::TopNNode>(
+        nextPlanNodeId(),
+        sortingKeys,
+        sortingOrders,
+        (int32_t)fetchRel.count(),
+        false /*isPartial*/,
```

Comment:
Would you update the PR description to mention the limitation of not converting 'partial' flags for all these plan nodes?